docs/docs/filename.md

```diff
@@ -72,19 +72,19 @@ Note that here, each slash (`/`) corresponds to a different directory:
 * `authorid`: the ID of the author of the image.
 * `date`: the date the image was posted on the server, using the format "MM-dd-yyyy HH.mm".
 * `all`: all tags.
-* `allo`: do not use this unless you know what you're doing. It is the same as `%all%`, but without the removal of forbidden characters and any other kind treatment (using it as a filename may cause some images to not save).
-* `source`: the source of this file, usually an URL.
+* `allo`: do not use this unless you know what you're doing. It is the same as `%all%`, but without the removal of forbidden characters and any other kind of treatment (using it as a filename may cause some images to not save).
+* `source`: the source of this file, usually a URL.
 * `sources`: all sources for this file, usually a list of URLs.
 * `url_file`: the URL of that file.
 * `url_original`: the URL of that file's original.
 * `url_sample`: the URL of that file's sample.
 * `url_thumbnail`: the URL of that file's thumbnail.
 * `url_page`: the URL of the details page of that file.
-* `count`: counts the current image downloaded during a batch download. Useful to have unique counters per batches.
-* `num`: counter that goes up for every file with the same name. Useful to prevent overwriting already existing files.
+* `count`: counts the current image downloaded during a batch download. Useful to have unique counters per batch.
+* `num`: a counter that goes up for every file with the same name. Useful to prevent overwriting already existing files.
 * `position`: the position of the image in the results list or gallery (starts at 1).
 * `old_directory`: when renaming, store the local path for the current file. (useful to rename images, without changing directory structure).
-* `old_filename`: when renaming, store the local filename for the current file. (useful to move images, but keeping their local names) Beware, this token keep the extension of the file, therefore, should not be used with %ext%.
+* `old_filename`: when renaming, store the local filename for the current file. (useful to move images, but keeping their local names) Beware, this token supports the extension of the file, therefore, should not be used with %ext%.
```

Comment:
"supports" sounds a bit weird here since the token always contains the extension, maybe "contains" could convey this meaning while being better than "keep"? 🤔

Reply:
You're absolutely right. "Keep" might be a bit confusing, and "contains" indeed conveys the intended meaning better.